hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/AttributeSearchService.kt

```diff
@@ -16,6 +16,10 @@ class AttributeSearchService(
     log.info("searchByAttributes called with request: $request, pageable: $pageable")
   }
 
+  fun getAttributes() = attributes.map { it.key to it.value.toString().lastWord() }.toMap()
```

Comment:
why map the last word?

Reply:
Because otherwise I get things lke `class kotlin.String`

Reply:
The endpoint is only really for developers to see what they can search on - I don't envisage it being used by clients to validate types etc.